src/JuMP.jl

```diff
@@ -470,6 +468,30 @@ function unset_silent(model::Model)
     return MOI.set(model, MOI.Silent(), false)
 end
 
+"""
+    writeLP(model::Model, fname::AbstractString)
```

Comment:
The analogy to `write` would also suggest providing a version that writes to an `IO` object.